### PR TITLE
Missing Django 3.1 items

### PR DESCRIPTION
**Pull request by mkurnikov**
Closes https://github.com/typeddjango/django-stubs/pull/556

### CODE REVIEW

**Comment by sobolevn** [comment]
@mkurnikov have you seen our discussion in #439 

Glad to see you 🙂

**Comment by mkurnikov** [comment]
For the PR this comment https://github.com/typeddjango/django-stubs/issues/439#issuecomment-670140107 is valid, it doesn't break anything. 

For the general case, I see this comment https://github.com/typeddjango/django-stubs/issues/439#issuecomment-707328556, is there anything done in that direction that I can look at?

**Comment by sobolevn** [comment]
> For the PR this comment #439 (comment) is valid, it doesn't break anything.

It might be a good way to go, because more and more users are reporting this. `3.1` is more and more used.

> For the general case, I see this comment #439 (comment), is there anything done in that direction that I can look at?

No, this requires a lot of work, I didn't have the time for it.

File: django-stubs/db/models/fields/json.pyi [hunk 1]
```diff
@@ -2,103 +2,43 @@ from . import Field
...
         **kwargs: Any
     ) -> None: ...
-    def check(self, **kwargs: Any): ...
-    def deconstruct(self): ...
-    def from_db_value(self, value: Any, expression: Any, connection: Any): ...
-    def get_internal_type(self): ...
-    def get_prep_value(self, value: Any): ...
-    def get_transform(self, name: Any): ...
-    def validate(self, value: Any, model_instance: Any) -> None: ...
     def value_to_string(self, obj: Any): ...
```
Comment (sobolevn) [inline django-stubs/db/models/fields/json.pyi:17]: `-> str`

**Pull request merged**